prestashop/scripts/init-and-run.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -x
+
+if [ ! -f /var/www/html/prestashop/.installed ]; then
+  
+  until mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" -h"${MYSQL_HOST}"; do
```

Comment:
This should be done in the docker-compose using a health-check https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y